### PR TITLE
Feature/piavlo/prevent intoify overflow and limit max batches in inotify processing loop

### PR DESCRIPTION
**Pull request by piavka**

### CODE REVIEW

File: bin/tailf2kafka [hunk 1]
```diff
@@ -87,10 +88,11 @@ end
 @files = {}
 @threads = {}
 @position_file = @settings[:tailf][:position_file]
-@flush_interval = @settings[:tailf][:flush_interval]
-@max_batch_lines = @settings[:tailf][:max_batch_lines]
+@flush_interval = @settings[:tailf].has_key?(:flush_interval) ? @settings[:tailf][:flush_interval] : 1
+@max_batch_lines = @settings[:tailf].has_key?(:max_batch_lines) ? @settings[:tailf][:max_batch_lines] : 1024
```
Comment (felka) [inline bin/tailf2kafka:92]: don't you think 1024 is a lot of lines? Some files are getting low volumes, like postback events is about 100 lines per minute?

Comment (felka) [inline bin/tailf2kafka:92]: btw i see gem files here? we do not need them here....

Comment (piavlo) [inline bin/tailf2kafka:92]: that's why i removed them :)
ilya added them in previous PR for osme reason

Comment (piavlo) [inline bin/tailf2kafka:92]: a lot of lines for what? this is an upper limit not a lower limit

**Pull request merged**